snmp/datadog_checks/snmp/data/profiles/_generic-bgp4.yaml

```
@@ -6,6 +6,8 @@ metrics:
       name: bgpPeerTable
       OID: 1.3.6.1.2.1.15.3
     symbols:
+      - name: peerConnection
```

```suggestion
      - name: peerConnectionByState
```
What do you think of something like this? Might be more clear for a user how to use this metric.

---

True, thanks!